test/test_helper/workspace.bash

```diff
@@ -9,10 +9,23 @@ function add_new_workspace() {
 	echo "${TMP_DIR}"
 }
 
+function add_dir_with_ws() {
+	TEST_DIR="$1"
+	TMP_DIR=$(mktemp -d)
+	cp -r "$TEST_DIR"/* "$TMP_DIR"
+
+	YAML="$TMP_DIR"/"$2"
+	WS_DIR=$(dirname "$YAML")
```

Comment:
```suggestion
	WS_DIR="$TMP_DIR"
```

Reply:
ignore this one, I see that difference now

Reply:
Good catch, left over from fixups.

Reply:
if $2 has a sub dir it'll be different.. actually you probably want it to not act like that right?

Reply:
either or, approved.. change if u want